README.rst

```diff
@@ -9,6 +9,12 @@ p4c-bm
 Generates the JSON configuration for the `behavioral-model (bmv2)
 <https://github.com/p4lang/behavioral-model>`_.
 
+NOTE: This repository is no longer under active development or
```

Comment:
Feel free to put this whole paragraph in bold to emphasize it...

Reply:
done